webapp/src/components/react_select_setting.jsx

```diff
@@ -53,7 +64,7 @@ export default class ReactSelectSetting extends React.PureComponent {
         const valid = Boolean(this.props.value);
         this.setState({invalid: !valid});
         return valid;
-    };
+    }
```

Comment:
super nitpick: can you change your IDE settings to prefer semicolons at the end of arrow functions? (it's "terminate statements with semicolons" in mine.)  For now -- we can discuss if we want our team style to be no semicolons over the offsite. :)

Reply:
Added. This is a convention established throughout the webapp repo as well.